### PR TITLE
Add information related to the V3 API ML Office Hours. 

### PR DESCRIPTION
**Pull request by balhar-jakub**

### CODE REVIEW

File: _data/vnext_office_hours.yml [hunk 1]
```diff
@@ -22,5 +22,5 @@ extenders:
   - date: 10/06/2022 8AM - 9AM ET
```
Comment (jp669844) [inline _data/vnext_office_hours.yml:22]: Should not be 10/06/2023?

File: _data/projects.yml [hunk 2]
```diff
@@ -47,6 +47,22 @@
   img_url: assets/img/nodejs-sdk-project.webp
   stage: Generally Available
   dedication: Core
+- name: Jobs
+  description: Provides APIs to handle jobs access. Use the z/OSMF directly as it provides richer functionality.
+  github_url: https://github.com/zowe/jobs
+  slack_url: https://github.com/zowe/api-layer
```
Comment (jp669844) [inline _data/projects.yml:53]: How about a direct Slack link? (https://openmainframeproject.slack.com/archives/CC5UUL005)

Comment (balhar-jakub) [reply _data/projects.yml:53]: Thanks for catching that. Fixed now

File: _data/vnext_office_hours.yml (hunk 1 above)
Comment (balhar-jakub) [reply _data/vnext_office_hours.yml:22]: Tahnks for catching. Also fixed for other office hours.

**Review by jp669844 — APPROVED** [review]

**Review by jp669844 — APPROVED** [review]

**Review by jp669844 — APPROVED** [review]

**Pull request merged**